src/Frontend/Themes/Fork/Core/Layout/Sass/components/_cookiebar.scss

```diff
@@ -0,0 +1,3 @@
+#cookieBar {
```

Comment:
Maybe it's time to change this to `#cookiebar` or `#cookie-bar` as we don't use the camelcased id's and classes anymore.